lib/config_cat/client.ex

```diff
@@ -202,31 +203,35 @@ defmodule ConfigCat.Client do
   defp evaluate(key, user, default_value, default_variation_id, %State{} = state) do
     user = if user != nil, do: user, else: state.default_user
 
-    case cached_settings(state) do
-      {:ok, settings, fetch_time_ms} ->
-        %EvaluationDetails{} =
-          details = Rollout.evaluate(key, user, default_value, default_variation_id, settings)
-
-        fetch_time =
-          case FetchTime.to_datetime(fetch_time_ms) do
-            {:ok, %DateTime{} = dt} -> dt
-            _ -> nil
-          end
-
-        %{details | fetch_time: fetch_time}
-
-      _ ->
-        message =
-          "Config JSON is not present when evaluating setting '#{key}'. Returning the `default_value` parameter that you specified in your application: '#{default_value}'."
-
-        EvaluationDetails.new(
-          default_value?: true,
-          error: message,
-          key: key,
-          value: default_value,
-          variation_id: default_variation_id
-        )
-    end
+    details =
+      case cached_settings(state) do
+        {:ok, settings, fetch_time_ms} ->
+          %EvaluationDetails{} =
+            details = Rollout.evaluate(key, user, default_value, default_variation_id, settings)
+
+          fetch_time =
+            case FetchTime.to_datetime(fetch_time_ms) do
+              {:ok, %DateTime{} = dt} -> dt
+              _ -> nil
+            end
+
+          %{details | fetch_time: fetch_time}
+
+        _ ->
+          message =
+            "Config JSON is not present when evaluating setting '#{key}'. Returning the `default_value` parameter that you specified in your application: '#{default_value}'."
+
+          EvaluationDetails.new(
+            default_value?: true,
+            error: message,
+            key: key,
+            value: default_value,
```

Comment:
Could you please add user in error case too. It seems this is a bug in the Ruby SDK I'll fix it there too.
```suggestion
            user: user,
            value: default_value,
```

Reply:
Could you please add the user to the details object in the `def evaluate(key, user, default_value, default_variation_id, settings)` too? Unfortunately, I can't see this file in this PR and can't give a suggestion for that.

Reply:
Done in 485f346